packages/nftables.rb

```diff
@@ -1,26 +1,26 @@
 # Adapted from Arch Linux nftables PKGBUILD at:
 # https://github.com/archlinux/svntogit-packages/raw/packages/nftables/trunk/PKGBUILD
 
-require 'package'
+require 'buildsystems/autotools'
 
-class Nftables < Package
+class Nftables < Autotools
   description 'Netfilter tables userspace tools'
   homepage 'https://netfilter.org/projects/nftables/'
-  version '1.0.8'
+  version '1.0.9'
```

Comment:
We should tag this with `-py3.12`...

Reply:
How do we include the python files?  Do we need to update the package?